packages/remix-server-adapter/src/defaultRemixConfig.ts

```diff
@@ -0,0 +1,8 @@
+import type { AppConfig } from '@remix-run/dev'
+
+export const config: AppConfig = {
+  serverBuildTarget: 'deno',
```

Comment:
The Remix team mentioned they're phasing out `serverBuildTarget`, but for the time being it remains.

Reply:
```suggestion
  serverBuildTarget: 'deno',
  serverModuleFormat: 'esm',
  serverPlatform: 'neutral',
```
We could add these to be ready for the deprecation

Reply:
I just pushed that. Didn't see you comment. 😅 

These are set when the `serverBuildTarget` is `deno`, but since Remix is phasing out `serverBuildTarget` soon as @ascorbic mentions, I'm adding these explicitly.